src_c/pgcompat.c

```diff
@@ -43,3 +43,13 @@ SDL_CreateRGBSurfaceWithFormat(Uint32 flags, int width, int height, int depth,
 }
 
 #endif /*if !(SDL_VERSION_ATLEAST(2, 0, 5))*/
+
+#if !(SDL_VERSION_ATLEAST(2, 0, 9))
+int
+SDL_GameControllerRumble(void *gamecontroller, Uint16 low_frequency_rumble,
+                         Uint16 high_frequency_rumble, Uint32 duration_ms)
+{
+    SDL_SetError("SDL 2.0.9 or greater required");
```

Comment:
```suggestion
    SDL_SetError("SDL 2.0.9 or greater required");
    return -1;
```
To set the correct error code that function calls to this will expect